docs/source/code_repo.rst

```diff
@@ -6,6 +6,49 @@ DoWhy is hosted on GitHub.
 You can browse the code in a html-friendly format `here
 <https://github.com/Microsoft/dowhy>`_.
 
+v0.9: New functional API (preview), faster refutations, and better independence tests for GCMs
+----------------------------------------------------------------------------------------------
+December 5 2022
+
+* Preview for the new functional API (see `notebook <https://github.com/py-why/dowhy/blob/main/docs/source/example_notebooks/dowhy_functional_api.ipynb>`_). The new API (in experimental stage) allows for a modular use of the different functionalities and includes separate fit and estimate methods for causal estimators. Please leave your feedback here. The old DoWhy API based on CausalModel should work as before.  (@andresmor-ms)
+
+* Faster, better sensitivity analyses. 
+    * Many refutations now support joblib for parallel processing and show a progress bar (@astoeffelbauer, @yemaedahrav).
+    * Non-linear sensitivity analysis [`Chernozhukov, Cinelli, Newey, Sharma & Syrgkanis (2021) <https://arxiv.org/abs/2112.13398>`_, `example notebook <https://github.com/py-why/dowhy/blob/main/docs/source/example_notebooks/sensitivity_analysis_nonparametric_estimators.ipynb>`_] (@anusha0409)
+    * E-value sensitivity analysis [`Ding & Vanderweele (2016) <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4820664/>`, `example notebook <https://github.com/py-why/dowhy/blob/main/docs/source/example_notebooks/sensitivity_analysis_testing.ipynb>`_] (@jlgleason)
+
+* Better conditional independence tests through the causal-learn package (@bloebp)
+
+* Algorithms for computing efficient backdoor sets [`example notebook <https://github.com/py-why/dowhy/blob/main/docs/source/example_notebooks/dowhy_efficient_backdoor_example.ipynb>`_] (@esmucler)
+
+* Support for estimating controlled direct effect (@amit-sharma)
+
+* Support for multi-valued treatments for econml estimators (@EgorKraevTransferwise)
+
+* Revamped docs to provide a task-based view of dowhy's functionalities. Also, you can now browse docs separately for different versions of DoWhy (@petergtz)
```

Comment:
> Also, you can now browse docs separately for different versions of DoWhy 

This was already available in the previous version, so you can probably remove that.

How about:
```
* New PyData theme for documentation with new homepage, Getting started guide, revised User Guide and examples page (@petergtz)
```

The task-based view is not really ready yet. I've got only a draft of that, but it's not ready for sharing just yet.

Reply:
Got it, making those changes.